internal/cmd/flink/command_compute_pool_update.go

```diff
@@ -0,0 +1,51 @@
+package flink
+
+import (
+	"github.com/spf13/cobra"
+
+	flinkv2 "github.com/confluentinc/ccloud-sdk-go-v2-internal/flink/v2"
+	pcmd "github.com/confluentinc/cli/internal/pkg/cmd"
+	"github.com/confluentinc/cli/internal/pkg/output"
+)
+
+func (c *command) newComputePoolUpdateCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "update <id>",
+		Short: "Update a Flink compute pool.",
+		Args:  cobra.ExactArgs(1),
+		RunE:  c.computePoolUpdate,
+	}
+
+	pcmd.AddEnvironmentFlag(cmd, c.AuthenticatedCLICommand)
+	pcmd.AddOutputFlag(cmd)
+
+	return cmd
+}
+
+func (c *command) computePoolUpdate(cmd *cobra.Command, args []string) error {
+	id := c.Context.GetCurrentFlinkComputePool()
+	if len(args) > 0 {
+		id = args[0]
+	}
+
+	update := flinkv2.FcpmV2ComputePoolUpdate{
+		Id: flinkv2.PtrString(id),
+		Spec: &flinkv2.FcpmV2ComputePoolSpecUpdate{
+			MaxCfu: flinkv2.PtrInt32(1),
```

Comment:
what this `update` command doing? Looks like it's always setting max cfu to 1

Reply:
Oops!